modules/kubernetes_node_pool/gke_node_pool/0.1/facets.yaml

```
@@ -58,27 +64,30 @@ spec:
-    - taints
-    - labels
+  - instance_type
+  - min_node_count
+  - max_node_count
+  - disk_size
+  - taints
+  - labels
 sample:
   flavor: gke_node_pool
-  $schema: https://facets-cloud.github.io/facets-schemas/schemas/kubernetes_node_pool/kubernetes_node_pool.schema.json
+  $schema: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)